src/background/stores/assessment-store.ts

```diff
@@ -54,10 +54,10 @@ export class AssessmentStore extends PersistentStore<AssessmentStoreData> {
         private readonly assessmentDataConverter: AssessmentDataConverter,
         private readonly assessmentDataRemover: AssessmentDataRemover,
         private readonly assessmentsProvider: AssessmentsProvider,
-        protected readonly idbInstance: IndexedDBAPI,
-        protected readonly persistedData: AssessmentStoreData,
+        idbInstance: IndexedDBAPI,
```

Comment:
Do we actually need to remove `protected readonly` from all of these parameter properties?

Reply:
No, it's not functionally necessary, but I noticed that they weren't being used anywhere other than in this constructor so I figured it would make sense to minimize their scope.

Reply:
Makes sense - I'd only looked at changed usage outside the constructor, and missed that there was no existing need for these to be properties before the change. Thanks!